src/data/store.test.js

```diff
@@ -60,17 +60,17 @@ describe('store aggregator module', () => {
         });
       });
       describe('if no SEGMENT_KEY', () => {
-        const key = configuration.SEGMENT_KEY;
+        const key = getConfig().SEGMENT_KEY;
```

Comment:
I would've thought we'd need a `let key;` in the scope above this one for the variable to persist until `afterEach()`, but... this did work before, right? :)

Reply:
It did work! My understanding is that the anonymous function in `afterEach(...)` creates a closure that captures `key`, ensuring that the value of `key` will be accessible as long as the anonymous function exists.